src/scss/grommet-core/_objects.quote.scss

```diff
@@ -1,6 +1,7 @@
 .quote {
   border-width: $inuit-base-spacing-unit;
   border-style: solid;
+  max-width: calc(100vw - #{$inuit-base-spacing-unit * 3});
```

Comment:
No. 100% will not work correctly.

The spacing is to account for the space around the Quote, however I have updated it to use double instead.